src/Slider/Slider.js

```diff
@@ -44,31 +44,60 @@ export default function Slider({
   } else {
     marks = false;
   }
+  // prevent horizontal keyboard navigation for vertical slider
+  const preventHorizontalKeyboardNavigation = (event, orientation) => {
+    if (orientation === "vertical") {
+      if (event.key === "ArrowLeft" || event.key === "ArrowRight") {
+        event.preventDefault();
+      }
+    }
+  };
+  // set label position based on orientation
+  let style = {};
+  if (orientation === "horizontal") {
+    style = {
```

Comment:
If ArrowLeft and ArrowRight are disabled for a vertical slider should the up and down arrows be disabled for the horizontal slider?